views/dashboard.jade

```diff
@@ -72,7 +72,7 @@ block content
   .col-lg-6.col-lg-offset-3
     .panel.panel-default
       .panel-body#t
-        .editable.textBox(tabindex="1") Enter your tweetstorm here.. Remember, no character limits.
+        .editable.textBox(tabindex="1") Enter your tweetstorm here.. 1000 character max --> that's ~27^100 possible texts, or more than all the atoms in all 7 multiverses
```

Comment:
Do we really need the `27^100` part? We could try adding something more fun instead. Maybe a random prompt?